src/components/FormTextarea.tsx

```diff
@@ -17,6 +17,7 @@ interface FormTextareaProps {
     disabled?: boolean;
     style?: React.CSSProperties;
     flat?: 'top' | 'bottom' | 'both';
+    minHeight?: number;
```

Comment:
Why not with `styled`?

Reply:
Right, not a good approach 
Change to `StyledFormTextarea` in `Commnet`